README.md

```diff
@@ -1,18 +1,51 @@
 # Expression Atlas
 
-## Requirements
-- Docker v19+
-- Docker Compose v1.25+
-- 100 GB of available storage (experiment files, PostgreSQL and Solr backup snapshots and Docker volumes)
+## Prepare your development environment
 
-Notice that PostgreSQL and Solr snapshots are [`bind` mounted](https://docs.docker.com/storage/bind-mounts/) in order
-to move data back and forth from the containers. Actual files managed by either Solr or PostgreSQL are kept in volumes
-which will be reused even if the containers are removed or brought down by Docker Compose. If you want to start afresh
-delete the old volume (e.g. for Postgres `docker volume rm gxa-pgdata`) and re-run the necessary step to return to the
-initial state.
+### TL;DR
+```bash
+./docker/prepare-dev-environment/gradle-cache/run.sh -l gradle-cache.log && \
+./docker/prepare-dev-environment/volumes/run.sh -l volumes.log && \
+./docker/prepare-dev-environment/postgres/run.sh -l pg.log && \
+./docker/prepare-dev-environment/solr/run.sh -l solr.log
+```
 
-## Code
-Clone the repository of Bulk Expression Atlas proper:
+### Requirements
+- Docker v19+
+- Docker Compose v1.25+
+- 100 GB of available storage for the following Docker volumes:
+    - Experiment files
+    - Bioentity properties (i.e. gene annotations)
+    - PostgreSQL
+    - SolrCloud and ZooKeeper
+    - Tomcat configuration files
```

Comment:
I’d only add something like:
> - 150 GB of temporary storage to write the files indexed by Solr

Reply:
We could add it later when everybody or at least one more people tested it.